src/server/pages/home-page.jsx

```diff
@@ -24,15 +29,67 @@ const LetsGetStarted = () => (
 
 export default class HomePage extends React.Component {
   render() {
+    const isLoggedIn = 'name' in this.props.userInfo
     const { notebookList } = this.props
     return (
       <div>
         <Header userInfo={this.props.userInfo} />
         <PageBody>
-          {notebookList.length ?
-            <TrendingNotebooksPage notebookList={notebookList} /> :
-            <LetsGetStarted />
+          <TopContainer>
+            {!isLoggedIn && <MarketingCopySplash /> }
+            {isLoggedIn && <LoggedInSplash userInfo={this.props.userInfo} />}
+            <PageHeader>Try These Examples</PageHeader>
+            <NotebookDisplay>
+              <NotebookDisplayItem
+                title="A Brief Tour through Iodide"
+                description="
+                   A tutorial that walks through all the important parts of Iodide."
+                href="https://extremely-alpha.iodide.io/notebooks/154/"
+                imageSource="https://media.giphy.com/media/5qF68SjjIT6khDkS5T/giphy.gif"
+              />
+              <NotebookDisplayItem
+                title="The Lorenz Attractor Up-Close"
+                description="
+                   A concise example demonstrating how powerful
+                   a web tech-focused notebook environment is for computational presentations."
+                href="https://extremely-alpha.iodide.io/notebooks/34/?viewMode=report"
+                imageSource="https://media.giphy.com/media/ftdkB78fuQ1Eb3J2o1/giphy.gif"
+              />
+              <NotebookDisplayItem
+                title="Pyodide: Scientific Python in your Browser"
+                description="
+                A tutorial demonstrating how
+                to use Python, Numpy, Pandas, and Matplotlib entirely within your browser."
```

Comment:
"how to integrate Python, Numpy, Pandas and Matplotlib with Web technologies"...?